design/cstor/v1/cstorpool.md

```diff
@@ -0,0 +1,238 @@
+# Proposed v1 schema for CSPC and CSPI
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Goals](#goals)
+* [CSPC and CSPI Schema Proposal](#cspc-schema-proposal)
```

Comment:
n_: link is not working. `#cspc-and-cspi-schema-proposal`

Reply:
There was a problem for 3rd bullet. Fixed it.